### ISSUE TITLE
Must check for non-existing git commit file

### ISSUE THREAD

**Issue by fatso83** [fatso83/check-commit-msg#1]
```


17:06	Commit failed with error
			0 files committed, 4 files failed to commit: NIM-1019 Add first test for publication MyDraftNotes
			husky > npm run -s precommit (node v11.11.0)
			
			No staged files match any of provided globs.
			
			
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․,․․․․․․․․․․․․․․․․․․․
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
			․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
			
			444 passing (2s)
			1 pending
			
			husky > npm run -s commitmsg (node v11.11.0)
			
			fs.js:119
			throw err;
			^
			
			Error: ENOENT: no such file or directory, open '.git/COMMIT_EDITMSG'
			at Object.openSync (fs.js:448:3)
			at Object.readFileSync (fs.js:348:35)
			at Object.<anonymous> (/home/carlerik/dev/nimble/frontend/node_modules/@fatso83/check-commit-msg/husky.js:12:22)
			... (show balloon)
```

**Comment by fatso83** [comment]
Could be related to #2

**Comment by fatso83** [comment]
Assuming fixed through #2
